tests/php/Authenticator/LoginHandlerTest.php

```diff
@@ -38,6 +39,8 @@ protected function setUp()
         parent::setUp();
         Config::modify()->set(MethodRegistry::class, 'methods', [Method::class]);
 
+        EnforcementManager::config()->set('enabled', true);
```

Comment:
I've left this in here because it makes the test more robust, even though it's strictly not required by the PR (since it no longer changes the default)